x-pack/docs/en/rest-api/security/update-api-key.asciidoc

```diff
@@ -1,5 +1,270 @@
 [role="xpack"]
 [[security-api-update-api-key]]
-=== Update API key information API
+=== Update API key API
 
-coming::[8.4.0]
+++++
+<titleabbrev>Update API key</titleabbrev>
+++++
+
+[[security-api-update-api-key-request]]
+==== {api-request-title}
+
+`PUT /_security/api_key/<id>`
+
+[[security-api-update-api-key-prereqs]]
+==== {api-prereq-title}
+
+* To use this API, you must have at least the `manage_own_api_key` cluster privilege.
+Users can only update their own API keys, i.e., API keys they created.
+To update another user's API key, you must use the run as feature.
+For more information, see <<run-as-privilege>>.
+
+IMPORTANT: It's not possible to use an API key as the authentication credential for this API.
+To update an API key, user password or <<security-api-get-token, token credentials>> are required.
+
+[[security-api-update-api-key-desc]]
+==== {api-description-title}
+
+The API keys created by <<security-api-create-api-key,create API Key>> or <<security-api-grant-api-key,grant API Key>> can be updated using this API.
+It's not possible to update expired API keys, or API keys that have been invalidated by <<security-api-invalidate-api-key,invalidate API Key>>.
+
+This API supports updates to an API key's access scope, i.e., its permissions, and its metadata.
+
+IMPORTANT: A call to <<security-api-update-api-key,update API Key>> may result in a change in the API key's access scope, even if you don't specify new <<security-api-update-api-key-api-key-role-descriptors,`role_descriptors`>> in the request.
+This is because the API key's access scope is <<security-api-update-api-key-api-key-role-descriptors,derived and updated>> based on a snapshot of the authenticated user's permissions at the time of the request.
```

Comment:
I understand `authenticated user` is what we use in the create-api-key page. But I don't think we should stick to it. The "owner user" is what we currently settled with and its usage should be promoted.